orcid-pub-web/src/main/java/org/orcid/api/filters/ApiRateLimitFilter.java

```diff
@@ -98,6 +98,9 @@ public class ApiRateLimitFilter extends OncePerRequestFilter {
 
     @Value("${org.orcid.papi.rate.limit.ip.whiteSpaceSeparatedWhiteList:127.0.0.1}")
     private String papiWhiteSpaceSeparatedWhiteList;
+    
+    @Value("${org.orcid.papi.rate.limit.clientId.whiteSpaceSeparatedWhiteList}")
+    private String papiClientIdWhiteSpaceSeparatedWhiteList;
```

Comment:
Lets make this a List and create a constructor that receive the whiteSpaceSeparatedWhiteList param and transform it into a list.

Reply:
Done in the afterProperties method that gets called just once in the lifecycle of the GenericFilterBean.